src/rules/no-quoted-strings-inside-quote-function/README.md

```diff
@@ -0,0 +1,46 @@
+# no-quoted-strings-inside-quote-function
```

Comment:
I think that the name should follow the same format as stylelint, e.g. `function-quote-no-quoted-strings-inside`:
https://github.com/stylelint/stylelint/blob/master/docs/user-guide/rules.md#function

Reply:
https://sass-lang.com/documentation/functions/string#quote